pyproject.toml

```diff
@@ -17,20 +17,20 @@ classifiers = [
     "License :: OSI Approved :: BSD License",
     "Programming Language :: Python :: 3",
     "Programming Language :: Python :: 3 :: Only",
-    "Programming Language :: Python :: 3.7",
     "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
     "Operating System :: OS Independent",
     "Topic :: Software Development :: Libraries :: Python Modules",
     "Typing :: Typed",
 ]
 
 dependencies = [
     "typing_extensions >= 4.0; python_version < '3.11'",
-    "dataclasses; python_version < '3.7'",
+    "dataclasses",
```

Comment:
This is wrong - the whole line can be removed.

Reply:
Please revert and do such cleanups in a different PR.

Reply:
can you please clarify. Unless I'm wrong, we still have `from dataclasses import dataclass` in _pages.py and _utils.py

Reply:
`dataclasses` was introduced in Python 3.7 via [PEP-557](https://peps.python.org/pep-0557/), and so can rely on the [builtin module](https://docs.python.org/3/library/dataclasses.html) for 3.7.